packages/mui-base/src/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,41 @@
+'use client';
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import type { BaseUIComponentProps } from '../utils/types';
+import { Field } from '../Field';
+
+/**
+ *
+ * Demos:
+ *
+ * - [Text Area](https://base-ui.netlify.app/components/react-text-area/)
+ *
+ * API:
+ *
+ * - [TextArea API](https://base-ui.netlify.app/components/react-text-area/#api-reference-TextArea)
+ */
+const TextArea = React.forwardRef(function TextArea(
+  props: TextArea.Props,
+  forwardedRef: React.ForwardedRef<HTMLTextAreaElement>,
+) {
+  return <Field.Control ref={forwardedRef} render={<textarea />} {...props} />;
```

Comment:
We should use the faster variant internally.

```suggestion
  return <Field.Control render={(fieldProps) => <textarea {...fieldProps} />} {...props} ref={forwardedRef} />;
```

Reply:
Resolving as `TextArea` is being removed, for now